src/test/java/pages/LoginPage.java

```diff
@@ -37,8 +37,7 @@ public boolean isDisplayMessageError() {
         return driver.findElement(ERROR_MESSAGE).isDisplayed();
     }
 
-    public boolean checkErrorText() {
-        String errorText = "Epic sadface: Username and password do not match any user in this service";
+    public boolean checkErrorText(String errorText) {
         return errorText.equals(driver.findElement(ERROR_MESSAGE).getText());
```

Comment:
Just make this method return String message like driver.findElement(ERROR_MESSAGE).getText() and them verify it in test via assert